core/src/main/java/com/xmartlabs/bigbang/core/helper/DateHelper.java

```diff
@@ -86,6 +86,16 @@ public static DateTimeFormatter getDeviceMediumDateFormat(Context context) {
   }
 
   /**
+   * Gets the android device short date format.
+   *
+   * @return {@link DateTimeFormatter} formatted the right way.
+   */
+  public static DateTimeFormatter getDeviceShortDateFormat(Context context) {
```

Comment:
maybe we should call it `getDeviceShortDateFormat`, what do you think @michaelEllisUy ?